src/pages/UserDashboard/Home.jsx

```diff
@@ -53,21 +54,17 @@ const Home = ({ userId, onCourseClick }) => {
   return (
     <div>
       <div className={styles.searchContainer}>
-        <form
-          id='submitSearchForm'
-          onSubmit={handleSearch}
-          className={styles.form}
-        >
+        <form id='submitSearchForm' className={styles.form}>
           <h2>Search courses:</h2>
           <input
             onChange={handleSearchTerm}
+            onClick={() => {
+              setSearchTerm('');
+            }}
             value={searchTerm}
             placeholder={searchTerm}
             className={styles.searchInput}
           />
-          <button className={styles.submit} type='submit'>
-            Search
-          </button>
         </form>
       </div>
       <div className={styles.coursesContainer}>
```

Comment:
I also made changes to the /userDashboard/Home page, so we need to be careful while merging.

Reply:
I resolved all conflicts with the current main branch, so it should be okay with my pull request as is.